packages/main/src/plugin/container-registry.ts

```diff
@@ -1030,8 +1030,10 @@ export class ContainerProviderRegistry {
     // eslint-disable-next-line @typescript-eslint/ban-types
     function onFinished(err: Error | null, output: {}) {
       if (err) {
+        eventCollect('finish', '' + err);
```

Comment:
eventCollect expects a string, i think you need to have `err.message` or whatever other prop you want to show.

Reply:
You don't like toString() on the error object ?

Reply:
The prob is that if it does not overwrite the default nodejs toString you would only see [Object object]. I don't know how to make it call the onFinished with an error but if you say it prints the info correctly, i trust you :+1:

Reply:
I can enhance to provide an Error object.

but yes it doesn't provide [Object object]

```bash
$ node -p "new Error('this is an error').toString()"
Error: this is an error
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/toString#using_tostring